lib/moneybird/service/sales_invoice.rb

```diff
@@ -7,6 +7,7 @@ class SalesInvoice
     include Moneybird::Traits::Save
     include Moneybird::Traits::Delete
     include Moneybird::Traits::Synchronization
+    include Moneybird::Traits::DownloadPdf
```

Comment:
Is this trait a shared by other invoice types/resources on the moneybird side?

Reply:
It could also be use at estimates. But this PR intended to cover only the download option at invoices.

Reply:
Ah cool, good to know.